### PR TITLE
Updates to references on how to use DB Browser

### LINKED ISSUE #238: Import Section - Directions outdated
The current version of DB SQLite doesn't have the tabs or organization described in the module. As a novice, without being able to actually follow the instructions, it's unclear what the purpose of the instructions are.

### PR DESCRIPTION
**Pull request by amyehodge**
Fixes #238 

I've updated everything I could find in this lesson throughout the modules that referenced the interface as it all appeared to still be about SQLite Manager and not the DB Browser that we are now using. Hopefully I caught everything.

### CODE REVIEW

File: _episodes/00-sql-introduction.md [hunk 1]
```diff
@@ -220,12 +219,12 @@ Finally, click **OK** one more time to confirm the operation.
 > - Import the `plots` and `species` tables
 {: .challenge}
 
-You can also use this same approach to append new data to an existing table.
+You can also use this same approach to append new fields to an existing table.
 
-## Adding data to existing tables
+## Adding fiels to existing tables
```
Comment (remram44) [inline _episodes/00-sql-introduction.md]: Typo :upside_down_face:

**Comment by amyehodge** [comment]
Ugh. That's what I get for being confident I remembered what the process was and not looking it up. I'll see if I can sort out how to do this correctly. Apologies.

**Comment by amyehodge** [comment]
Apologies, but I seem unable to follow the instructions in the Contributing document. This process seems to have changed significantly from the last time I did this. I don't understand how/where to create the branch. To create what I did above, I forked this repo, so maybe I need to start with cloning it to my desktop and creating a new branch? I am unable to create a branch at https://github.com/datacarpentry/sql-ecology-lesson in GitHub directly.

**Comment by amyehodge** [comment]
See #242

**Comment by remram44** [comment]
It's ok, Git is confusing enough, but fixing mistakes afterwards is downright ridiculous. I merged this as a382be53. Thanks again!

**Pull request closed without merging**